tests/test_tools_module.py

```
@@ -0,0 +1,34 @@
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+from mmrelay.tools import get_sample_config_path, get_service_template_path
+
+
+def test_get_sample_config_path_uses_package_resource_files() -> None:
+    package_root = Path("/tmp/fake-tools")
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake-tools"

(S108)